modules/stochastic_tools/examples/surrogates/gaussian_process/sub.i

```diff
@@ -0,0 +1,66 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 100
+  xmax = 0.03
+  elem_type = EDGE3
+[]
+
+[Variables]
+  [T]
+    order = SECOND
+    family = LAGRANGE
+  []
+[]
+
+[Kernels]
+  [diffusion]
+    type = MatDiffusion
+    variable = T
+    diffusivity = k
+  []
+  [source]
+    type = BodyForce
+    variable = T
+    value = 10000
+  []
+[]
+
+[Materials]
+  [conductivity]
+    type = GenericConstantMaterial
+    prop_names = k
+    prop_values = 5.0
+  []
+[]
+
+[BCs]
+  [right]
+    type = DirichletBC
+    variable = T
+    boundary = right
+    value = 300
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+[]
+
+[Postprocessors]
+  [avg]
+    type = AverageNodalVariableValue
+    variable = T
+  []
+  [max]
+    type = NodalExtremeValue
+    variable = T
+    value_type = max
+  []
+[]
+
+[Outputs]
+[]
```

Comment:
Need to add tests for all the inputs in the examples folder. MOOSE SQA says we can't check in input files without adding tests.

Reply:
Added tests. Attempted to model off example tests already in there (poly chaos). Marked full model test as heavy.